internal/bloblang/query/methods_geoip.go

```diff
@@ -0,0 +1,58 @@
+package query
+
+import (
+	"net"
+
+	"github.com/oschwald/geoip2-golang"
+)
+
+var _ = registerSimpleMethod(
+	NewMethodSpec(
+		"geoip_city", "",
+	).Param(ParamString("path", "Path to city mmdb file")),
+	func(args *ParsedParams) (simpleMethod, error) {
+		// TODO: Ensure path is a static. Unsure how to accomplish this
```

Comment:
Leave this one with me, I'll need to add a way to optionally memoize constructors for functions like this, and part of that will likely be destruction as well (in which we'll call `db.Close()`).

Reply:
It looks like this is done. 
stracing this PR but merged with df919d7e9c7c3689132a5cf80799d3b5d0290308 shows that the mmdb file is only read once.

Reply:
I think it's good enough for now to proceed, ideally I want to add some sort of decaying cache but I don't think we need to block this PR on that.

Reply:
Decaying would be nice but is really unnecessary for this use case imho.
Is the plan to wrap each geoip2 method into a bloblang method?

Reply:
I am ok with it, it's just to be sure how to implement the rest ;)